packages/blockchain-wallet-v4-frontend/src/components/TransactionListItem/index.js

```diff
@@ -11,7 +11,12 @@ class ListItemContainer extends React.PureComponent {
   state = { isToggled: false }
 
   handleToggle = () => {
+    const { coin, ethTxActions, erc20List, transaction } = this.props
+
     this.setState({ isToggled: !this.state.isToggled })
+    if (this.state.isToggled && includes(coin, erc20List)) {
+      ethTxActions.fetchTransaction(transaction.hash)
```

Comment:
https://gyazo.com/debb2711b27a95e968aa6c574e4a7f3f

Toggling transaction on PAX throws this error